app/assets/javascripts/backbone/views/log_modal_view.coffee

```diff
@@ -2,6 +2,7 @@ class MS.Views.LogModalView extends Backbone.View
 
   initialize: (options) ->
     @parentView = options.parentView
+    @timelineTableView = options.timelineTableView
```

Comment:
In please try to avoid adding references to other views like this. Consider this an anti-pattern. I know we have done this in the past but it tends to produce unmaintainable code. The better way to enable inter-view communication is by using callbacks and/or promises. We are already using callbacks with the ProjectStepModalView. Promises are really neat too. Check them out if you are not familiar with them.